tests/ImageTests.php

```diff
@@ -447,7 +460,12 @@ protected function output($file)
     public function setUp()
     {
         $dir = $this->output('');
-        `rm -rf $dir`;
+        
+        if (PHP_OS === 'WINNT') {
+            `rd /s /q "$dir"`;
```

Comment:
Even if it's a simple function, this library is not updated from a long time and don't have any test.

Not reliable for me, but I let @Gregwar take the decision.